API/server.js

```diff
@@ -28,19 +28,19 @@ function getRandomAnimalNo(){
 
 app.get('/', (req, res) => res.send('Hello World!'))
 
-app.get('/search/animal/:id', function(req, res){
+app.get('/search/animal/:id', function(req, res){ //
     // generate random number that equates to animal
-    getRandomAnimalNo();
+    getRandomAnimalNo(); //
```

Comment:
This route doesn't make use of the id supplied, by using `req.params.id` you can access the information passed through to the server.

By using `getRandomAnimalNo();` you always generate a random animal regardless of the query passed through.